alphagov/tsuru-terraform
# [#91207216] Add coreOS docker nodes

https://www.pivotaltracker.com/story/show/91207216 

**What**

This PR moves our docker nodes from running on Ubuntu 14.0.4 LTS  to CoreOS Stable channel (723.3.0) 

This pull request should be reviewed in the context of:

**Acceptance criteria**
**Given** a script (or set of instructions) for creating a Tsuru cluster
**And** that the infrastructure has already been provisioned
**When** I execute the script (or follow the instructions)
**Then** a docker node running on CoreOS is added to the Tsuru cluster and able to run applications

**How to test**

Check out this branch and deploy a new environment. Check for the number of coreos nodes in the platform of your choice. SSH into one of the coreos nodes and check that they are running the stable version of coreos. 

This PR only covers half of this story - there is a second PR on tsuru-ansible to be merged. 
https://github.com/alphagov/tsuru-ansible/pull/165

 **Who should review**

Anyone but @mtekel or @dhilton
